tests/unit/lib/models/ActionQueueEntry.spec.js

```diff
@@ -43,6 +43,25 @@ describe('ActionQueueEntry', () => {
             entry.getAttribute('details'), { roundTrip: true });
     });
 
+    it('should set an error when retrieving an unset attribute and ' +
+    '"required" option is set', () => {
+        const entry = ActionQueueEntry.create('sendPigeon')
+              .setAttribute('toCountry', 'Finland');
```

Comment:
🤔 Why is this pigeon being sent to Finland?

Reply:
Because it is pigeons' favorite country! It's also a good pigeon hub for long haul flights.